packages/tuono/src/build/index.ts

```diff
@@ -21,6 +21,8 @@ const VITE_SSR_PLUGINS: Array<Plugin> = [
        * @see https://github.com/tuono-labs/tuono/issues/218
        */
       MessageChannel: ['tuono/ssr', 'MessageChannelPolyfill'],
+      MessageEvent: ['tuono/ssr', 'MessageEventPolyfill'],
+      Event: ['tuono/ssr', 'EventPolyfill'],
```

Comment:
Do we need to create a task to port these polypill in rust in the future?

Reply:
I don't know but I would not think so because they just hold values...
I think it could be worth running an investigation in the future but not sure if addressing them makes sense. WDYT?

Reply:
Ok, no need to track this for now. We already too much work ahead of us 😂